.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+  "parserOptions": {
+      "ecmaVersion": 7,
```

Comment:
I think we want `es2015` since thats what `babel-preset-cf` uses right?
https://github.com/cloudflare/babel-preset-cf/blob/master/index.js